### PR TITLE
CNV-12270: Adding new storage feature to RN

### PR DESCRIPTION
**Pull request by bgaydosrh**
for 4.8 only.

Jira: https://issues.redhat.com/browse/CNV-12270

Tagging @aglitke for code review.
Tagging @duyanyan for QE review.

Direct doc preview link: https://deploy-preview-34236--osdocs.netlify.app/openshift-enterprise/latest/virt/virt-4-8-release-notes?utm_source=github&utm_campaign=bot_dp#virt-4-8-storage-new

### CODE REVIEW

File: virt/virt-4-8-release-notes.adoc [hunk 1]
```diff
@@ -72,7 +72,7 @@ The SVVP Certification applies to:
 
 //CNV-12269 When a VM frees space on a virtual disk the discard requests are passed to the underlying storage device which may free up real storage capacity.
 
-//CNV-12270 CDI can now automatically choose preferred accessMode and volumeMode settings when importing VM disk images.
+* The Conainerized Data Interface (CDI) now automatically chooses preferred `accessModes` and `volumeMode` settings when you import virtual machine disk images.
```
Comment (aglitke) [inline virt/virt-4-8-release-notes.adoc:75]: Containerized

We need to be a bit more measured with this note.  None of: the UI, virtctl, or VM Templates are using the 'storage' field of the DataVolume spec to activate this new behavior.  I think we should state that "a new way of specifying Data Volumes is available and, when used, will allow the system to optimize 'accessModes', 'volumeMode' and storage capacity when allocating storage.

Comment (bgaydosrh) [reply virt/virt-4-8-release-notes.adoc:75]: Hi @aglitke - is it possible to say this:

* You can now specify data volumes using the storage API. The storage API, unlike the PVC API, allows the system to optimize 'accessModes', 'volumeMode' and storage capacity when allocating storage.

I understand some of the generalization but I think mentioning the storage API is important. WDYT?

**Review by aglitke — APPROVED** [review]

**Comment by duyanyan** [comment]
LGTM

File: virt/virt-4-8-release-notes.adoc [hunk 2]
```diff
@@ -72,7 +72,7 @@ The SVVP Certification applies to:
 
 //CNV-12269 When a VM frees space on a virtual disk the discard requests are passed to the underlying storage device which may free up real storage capacity.
 
-//CNV-12270 CDI can now automatically choose preferred accessMode and volumeMode settings when importing VM disk images.
+* You can now specify data volumes using the storage API. The storage API, unlike the PVC API, allows the system to optimize 'accessModes', 'volumeMode' and storage capacity when allocating storage.
```
Comment (ousleyp) [inline virt/virt-4-8-release-notes.adoc:75]: s/using/by using

I think that accessModes and volumeMode should be in backticks (like you had them before).

There should be a comma after `volumeMode`

**Review by ousleyp — COMMENTED** [review]
overall this is looking good, except for a few nits

File: virt/virt-4-8-release-notes.adoc (hunk 2 above)
Comment (bgaydosrh) [reply virt/virt-4-8-release-notes.adoc]: Done

**Review by ousleyp — APPROVED** [review]
lgtm

**Pull request merged**